.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,49 @@
+# Adapted from sample vite workflow file: https://vitejs.dev/guide/static-deploy#github-pages
+name: deploy
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+# Sets the GITHUB_TOKEN permissions to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow one concurrent deployment
+concurrency:
+  group: 'pages'
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
should we really limit ourselves to 18 here? Pretty sure I'm running from 20 locally.

Reply:
No, you're right I think we can remove it.